test/UserCreatorTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+require_once "../vendor/autoload.php";
+use Icepace\UserCreator;
+use PHPUnit\Framework\TestCase;
+
+class UserCreatorTest extends TestCase
+{
+    public function testSuccessSanitiseUsername()
+    {
+        $input = 'gary ';
+        $expectedOutput = 'gary';
+        $actualOutput = UserCreator::sanitiseUsername($input);
+        $this::assertEquals($expectedOutput,$actualOutput);
+    }
+}
```

Comment:
need to delete this whole file because the method sanitiseUsername is private lol

Reply:
Deleted unused test.